modules/acid/src/main/ts/ephox/acid/gui/components/SaturationBrightnessPalette.ts

```diff
@@ -74,6 +75,7 @@ const paletteFactory = (_translate: (key: string) => string, getClass: (key: str
   const setPaletteThumb = (slider: AlloyComponent, hex: Hex): void => {
     const hsv = HsvColour.fromRgb(RgbaColour.fromHex(hex));
     Slider.setValue(slider, { x: hsv.saturation, y: 100 - hsv.value });
+    Attribute.set(slider.element, 'aria-valuetext', `Saturation ${hsv.saturation}%, Brightness ${hsv.value}%`);
```

Comment:
We need to pass these though I18n there is variables that you can use so can do this: https://fiddle.tiny.cloud/EGiaab

The component gets a translate function passed in that takes a key and returns a value that will not really work in this case. So it needs to take the key and the numbers to produce a full string maybe we can change the translate function to do an array instead for the color picker component that way you could pass N args like we do for the translate function in the backstage.

Reply:
Oh you'd already mentioned this. right.

Part of the complexity here is the component `translate` function actually maps to codes, because it was ported from TBIO verbatim, and it has a custom translate wrapper function that enforces strings.
https://github.com/tinymce/tinymce/blob/92f53fe893ca049293c02e5fe82a9ffa3056afc3/modules/tinymce/src/themes/silver/main/ts/ui/dialog/ColorPicker.ts#L11-L27